ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -109,12 +109,12 @@ ${listenerText}
    */
   get installBoundaryText() {
     const configPath = this.configFilePath || '<path>';
-    const ossContent = `curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add - ;\\
+    const ossContent = `curl -sS https://api.releases.hashicorp.com/v1/releases/boundary/latest | jq '.version';\\
```

Comment:
Why did we swap this out? This command was for adding the gpg key to be able to authenticate the repository package. We used apt to install with this command; not downloading a package to unzip